src/main/java/com/github/manolo8/darkbot/gui/utils/inspector/InspectorTree.java

```diff
@@ -61,8 +65,12 @@ public void treeWillCollapse(TreeExpansionEvent event) throws ExpandVetoExceptio
             public void ancestorAdded(AncestorEvent event) {
                 if (timer == null) {
                     timer = new Timer(delay, e -> {
-                        ((ObjectTreeNode) model.getRoot()).update(InspectorTree.this);
-                        invalidate();
+                        try {
+                            ((ObjectTreeNode) model.getRoot()).update(InspectorTree.this);
+                            invalidate();
+                        } catch (Throwable t) {
+                            t.printStackTrace();
```

Comment:
why is this throwing exceptions in the first place?

Reply:
due to memory misreads caused by invalid memory reads via different threads
anyway shouldn't happen anymore i hope

Reply:
if it doesn't happen, can we get rid of it?